packages/react-reconciler/src/ReactUpdateQueue.new.js

```diff
@@ -207,9 +202,6 @@ export function createUpdate(
 
     next: null,
   };
-  if (__DEV__) {
-    update.priority = getCurrentPriorityLevel();
```

Comment:
btw, are we actually using these in DevTools? I thought we added them for DevTools but I couldn't find any usage of them.

Reply:
It was added for the Suspense priority warning, which we later removed

Reply:
We don't need it in the new system because the priority can be determined from the lane without having to infer.

Reply:
I think the only place DevTools surfaces priority level info is in the Profiler, per commit:
https://github.com/facebook/react/blob/571f5ad2d695e6373ffbd8e86b27c11dad3d22cf/packages/react-reconciler/src/ReactFiberDevToolsHook.old.js#L85-L102

Maybe that's what you were thinking of?